README.md

```diff
@@ -80,10 +81,12 @@ To contribute, see [CONTRIBUTING.md](./CONTRIBUTING.md)
 - [mergestat](https://github.com/mergestat/mergestat) - Run SQL queries on git repositories.
 - [meteor](https://github.com/stefanlogue/meteor) - A highly customizable conventional commit message tool.
 - [mods](https://github.com/charmbracelet/mods) - AI on the CLI, built for pipelines.
+- [mult](https://github.com/dhth/mult) - lets you run a command multiple times and glance at the outputs via a TUI.
```

Comment:
```suggestion
- [mult](https://github.com/dhth/mult) - Run a command multiple times and glance at the outputs via a TUI.
```

Reply:
Ah, didn't see the latest build results. Changes pushed.